src/test/java/com/ericsson/ei/flowtests/TestConfigs.java

```diff
@@ -0,0 +1,106 @@
+import lombok.Getter;
+
+public class TestConfigs {
+
+    private static AMQPBrokerManager amqpBroker;
+    private static MongodForTestsFactory testsFactory;
+    private static ConnectionFactory cf;
+
+    final static Logger LOGGER = LoggerFactory.getLogger(TestConfigs.class);
+
+    @Getter
+    private static Connection conn;
+
+    @Getter
+    private static MongoClient mongoClient = null;
+
+    public static void init() throws Exception {
+        setUpMessageBus();
+        setUpEmbeddedMongo();
+    }
+
+    private static void setUpMessageBus() throws Exception {
+        LOGGER.debug("setting up message buss");
+        if (amqpBroker != null || conn != null || cf != null) {
+            return;
+        }
+
+        int port = SocketUtils.findAvailableTcpPort();
+        System.setProperty("rabbitmq.port", "" + port);
+        System.setProperty("rabbitmq.user", "guest");
+        System.setProperty("rabbitmq.password", "guest");
+        System.setProperty("waitlist.initialDelayResend", "500");
+        System.setProperty("waitlist.fixedRateResend", "100");
+        LOGGER.info("setting up message buss");
+        String config = "src/test/resources/configs/qpidConfig.json";
+        File qpidConfig = new File(config);
+        amqpBroker = new AMQPBrokerManager(qpidConfig.getAbsolutePath(), port);
+        amqpBroker.startBroker();
+        cf = new ConnectionFactory();
+        cf.setUsername("guest");
+        cf.setPassword("guest");
+
+        cf.setPort(port);
+        cf.setHandshakeTimeout(600000);
+        cf.setConnectionTimeout(600000);
+        conn = cf.newConnection();
+
+    }
+
+    public static MongoClient mongoClientInstance() throws Exception {
+        if (mongoClient == null) {
+            setUpEmbeddedMongo();
+        }
+
+        return mongoClient;
+    }
+
+    private static void setUpEmbeddedMongo() throws IOException {
+        if (mongoClient != null) {
+            return;
+        }
+
+        try {
+            testsFactory = MongodForTestsFactory.with(Version.V3_4_1);
+            mongoClient = testsFactory.newMongo();
+            String port = "" + mongoClient.getAddress().getPort();
+            System.setProperty("spring.data.mongodb.port", port);
+        } catch (Exception e) {
+            LOGGER.error(e.getMessage(), e);
+            e.printStackTrace();
+        }
+    }
+
+    public static void createExchange(final String exchangeName, final String queueName) {
+        final CachingConnectionFactory ccf = new CachingConnectionFactory(cf);
+        LOGGER.info("Creating exchange: {} and queue: {}", exchangeName, queueName);
+        RabbitAdmin admin = new RabbitAdmin(ccf);
+        Queue queue = new Queue(queueName, false);
+        admin.declareQueue(queue);
+        final TopicExchange exchange = new TopicExchange(exchangeName);
+        admin.declareExchange(exchange);
+        admin.declareBinding(BindingBuilder.bind(queue).to(exchange).with("#"));
+        ccf.destroy();
```

Comment:
There are a lot of things happening each line, please make blank rows to seperate it to more "logical blocks"